### PR TITLE
Added Code Portal Button

### PR DESCRIPTION
**Pull request by beichhor**
To provide more visibility to the Code Portal, a prominent button is displayed on the Millennium, Authorization, and SMART sections of the fhir.cerner.com site.

### CODE REVIEW

File: static/css/documentation.css [hunk 1]
```diff
@@ -218,6 +218,12 @@ a img {
   position: relative;
 }
 
+#header-button {
+  background: #5C9A32;
```
Comment (MaxPhilips) [inline static/css/documentation.css]: May want to use #7BC143

**Comment by MaxPhilips** [comment]
+1

File: static/css/documentation.css (hunk 1 above)
Comment (MaxPhilips) [reply static/css/documentation.css]: Hm, I'm wavering back to the original color.  I think it looks better a bit darker

Comment (beichhor) [reply static/css/documentation.css]: I think the original color definitely makes the text easier to read.

Comment (beichhor) [reply static/css/documentation.css]: I've gone ahead and dropped the change to revert it back to it's previous color.

File: Rules [hunk 2]
```diff
@@ -19,6 +19,7 @@
 preprocess do
   add_version_attribute
   add_solution_attribute
+  add_button_attribute
```
Comment (daliboz) [inline Rules]: May be simpler to just check @item[:solution] != 'soarian' in the header layout? That way, if we add other solutions, it's obvious when this piece depends on solution?

File: static/css/documentation.css (hunk 1 above)
Comment (daliboz) [reply static/css/documentation.css]: looks good - goal was to keep it consistent with the code site, and this color matches.

**Comment by daliboz** [comment]
+1 other than possibly simplifying to use existing attributes :)

File: Rules (hunk 2 above)
Comment (beichhor) [reply Rules]: The only issue with that is that the button will show up on the homepage. Not sure if that is expected behavior you would want.

Comment (daliboz) [reply Rules]: eh. probably not. Darn.

**Comment by daliboz** [comment]
Ok, just +1 :)

**Comment by shriniketsarkar** [comment]
This update was long overdue. I am glad we are getting this done :) Thanks @beichhor .

**Pull request merged**